src/linker/pipeline_schema.py

```diff
@@ -49,13 +50,13 @@ def _generate_schema(cls, name: str, *steps: Step) -> None:
             schema._add_step(step)
         return schema
 
-    @classmethod
-    def validate_input(cls, filepath: Path) -> Optional[List[str]]:
-        "Wrap the output file validator for now, since it is the same"
-        try:
-            validate_dummy_output(filepath)
-        except Exception as e:
-            return [e.args[0]]
+
+def validate_dummy_input(filepath: Path) -> Optional[List[str]]:
```

Comment:
I think this should not be hard-coded as dummy. And maybe make the validation function a dictionary for future flexibility